### ISSUE TITLE
vmalert is interrupted after a reload with alert rules.

### ISSUE THREAD

**Issue by stress-t** [VictoriaMetrics/VictoriaMetrics#480]
**Describe the bug**
vmalert is interrupted after a reload with alert rules.

**To Reproduce**
Send request to running vmalert
curl http://127.0.0.1:8880/-/reload

```
2020-05-14T07:55:32.294Z        info    VictoriaMetrics/app/vmalert/group.go:108        group "kubernetes-absent" started
2020-05-14T07:55:32.295Z        info    VictoriaMetrics/lib/httpserver/httpserver.go:77 pprof handlers are exposed at http://:8880/debug/pprof/
2020-05-14T07:55:40.268Z        info    VictoriaMetrics/app/vmalert/web.go:56   api config reload was called, sending sighup
2020-05-14T07:55:40.269Z        info    VictoriaMetrics/app/vmalert/main.go:107 SIGHUP received. Going to reload rules ["./vmalert/alerts/*.yaml"] ...
2020-05-14T07:55:40.269Z        info    VictoriaMetrics/app/vmalert/manager.go:77       reading alert rules configuration file from "./vmalert/alerts/*.yaml"
panic: runtime error: slice bounds out of range [13:11]

goroutine 394 [running]:
main.(*Group).updateWith(0xc00033eaa0, 0xc00017f840, 0x4, 0xc000108a50, 0x2c, 0xc0000b55e0, 0xd, 0xd, 0xc00001b200, 0xc00001b260)
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/group.go:63 +0x54f
main.(*manager).update(0xc0000b5880, 0x933f00, 0xc0000acf40, 0xc000085010, 0x1, 0x1, 0x1, 0x1, 0x1)
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/manager.go:100 +0x6b3
main.main.func2(0xc0000b5880, 0x933f00, 0xc0000acf40)
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/main.go:108 +0x256
created by main.main
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/main.go:99 +0x62f
```

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
./vmalert-prod --version
vmalert-20200513-185322-heads-master-0-g415b1ddf
```

**Used command-line flags**
--rule=./vmalert/alerts/*.yaml  --notifier.url=http://alertmanager:9093 --datasource.url 127.0.0.1:1111

**Comment by hagen1778** [comment]
Hi @stress-t! Could you pls check the updated version in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/a249cd9d22e0dea9e1f572147b62db7baaca21f8

Could you [build](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmalert#how-to-build-from-sources) `vmalert` from this commit and verify whether it works as expected?

**Comment by stress-t** [comment]
Now the reload is normal

**Comment by valyala** [comment]
The bugfix has been included in [v1.35.5](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
